_playwright-tests/UI/RedHatRepo.spec.ts

```diff
@@ -52,4 +52,4 @@ test.describe('Red Hat Repositories', () => {
             await page.getByTestId('snapshot_list_modal-ModalBoxCloseButton').click();
         });
     });
-});
```

Comment:
let's try avoid commiting these :) (I agree there should be trailing new line, but if someone commits it, usually just changing it is meh - but we can introduce lint for it)

Reply:
fixed 👍🏼